pkg/xds/topology/vip_outbounds.go

```diff
@@ -0,0 +1,50 @@
+package topology
+
+import (
+	"strings"
+
+	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+
+	mesh_proto "github.com/Kong/kuma/api/mesh/v1alpha1"
+	mesh_core "github.com/Kong/kuma/pkg/core/resources/apis/mesh"
+	"github.com/Kong/kuma/pkg/dns-server/resolver"
+)
+
+const VIPListenPort = uint32(80)
+
+func PatchDataplaneWithVIPOutbounds(dataplane *mesh_core.DataplaneResource,
+	dataplanes *mesh_core.DataplaneResourceList, resolver resolver.DNSResolver) (errs error) {
+	serviceVIPMap := map[string]string{}
+	for _, dp := range dataplanes.Items {
+		if dp.Meta.GetName() == dataplane.Meta.GetName() {
+			continue
+		}
+
+		for _, inbound := range dp.Spec.Networking.Inbound {
+			inService := inbound.GetTags()[mesh_proto.ServiceTag]
+
+			vip, err := resolver.ForwardLookup(inService)
+			if err != nil {
```

Comment:
This logic should be gone after we change `service.namespace.svc:port` to `service_namespace_svc_port` right?
Can you add todo to remove it?